packages/components/.storybook/manager.js

```diff
@@ -0,0 +1,11 @@
+import { addons } from "@storybook/addons"; // eslint-disable-line import/no-extraneous-dependencies
+
+addons.setConfig({
+  sidebar: {
+    // Show top-level directories (under components/src) as directories instead of separate
+    // sections. It's a little nicer seeing them as directories instead of their own sections.
+    // We only need this when auto-generating story titles. If we supply our own titles, we should
+    // delete this config.
+    showRoots: false,
```

Comment:
Without this, when using auto-generated titles we get each top-level directory as a separate section in storybook, which looks weird.

<img width="1015" alt="Screen Shot 2022-01-26 at 12 44 04 PM" src="https://user-images.githubusercontent.com/2503289/151244436-7da3ac23-78e4-4db7-a86f-b4a4ba8988da.png">

An alternative to this is the same thing mentioned in https://github.com/chanzuckerberg/edu-design-system/pull/813/files#r793032073.